pkg/kcp/nuke/loadResources.go

```diff
@@ -6,6 +6,7 @@ import (
 	cloudcontrolv1beta1 "github.com/kyma-project/cloud-manager/api/cloud-control/v1beta1"
 	"github.com/kyma-project/cloud-manager/pkg/common/actions/focal"
 	"github.com/kyma-project/cloud-manager/pkg/composed"
+	. "github.com/kyma-project/cloud-manager/pkg/kcp/nuke/types"
```

Comment:
This is confusing, please use `nuketypes.State` so it's clear when reading the code it's not the `State` from this package